webhook_to_fedora_messaging/models/apikey.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sqlalchemy import Column, Integer, UnicodeText, Boolean, ForeignKey
+from webhook_to_fedora_messaging.models.util import UUIDCreatableMixin, CreatableMixin, TZDateTime
+
+from webhook_to_fedora_messaging.database import Base
+
+from uuid import uuid4
+
+
+class APIKey(Base, UUIDCreatableMixin, CreatableMixin):
+    __tablename__ = "apikeys"
+
+    id = Column("id", Integer, primary_key=True, nullable=False)
```

Comment:
You don't need to repeat the column name as first argument of `Column`. This goes for the entire file and all others.